mne/dipole.py

```diff
@@ -43,4 +83,33 @@ def read_dip(fname, verbose=None):
     amplitude = data[:, 5]
     ori = data[:, 6:9]
     gof = data[:, 9]
-    return time, pos, amplitude, ori, gof
+    dipoles = dict(time=time, pos=pos, amplitude=amplitude, ori=ori,
+                   gof=gof, name=name)
```

Comment:
I am going to bed now and I probably won't get around to creating the class until next week sometime, so feel free to open a PR into my branch if you want